src/js/components/Grommet/__tests__/Grommet-test.js

```diff
@@ -15,35 +14,6 @@ const TestAnnouncer = ({ announce }) => {
   return <div>hi</div>;
 };
 
-const customBreakpointsTheme = {
-  global: {
-    deviceBreakpoints: {
-      phone: 'small',
-      tablet: 'medium',
-      computer: 'large',
-    },
-    breakpoints: {
-      small: {
-        value: 600,
-      },
-      medium: {
-        value: 800,
-      },
-      large: {
-        value: 1000,
-      },
-    },
-  },
-};
-
-const SSRTester = ({ ua }) => (
-  <Grommet theme={customBreakpointsTheme} userAgent={ua}>
-    <ResponsiveContext.Consumer>
-      {(size) => <Heading>{`Received size ${size} for ${ua}`}</Heading>}
```

Comment:
should we add this issue of testing to our end-to-end issue we have on the DS backlog? Just so we know when we start that we can account for this.

Reply:
Yeah I will add this to the end-to-end testing issue in our backlog